### PR TITLE
Portable emit tests

### PR DESCRIPTION
**Pull request by tmat**
Fix dev15.7.x to master merge issues.

### CODE REVIEW

File: src/Compilers/CSharp/Test/Emit/Attributes/InternalsVisibleToAndStrongNameTests.cs [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
-
+#if NET461
```
Comment (jaredpar) [inline src/Compilers/CSharp/Test/Emit/Attributes/InternalsVisibleToAndStrongNameTests.cs]: Why are all these `#if` necessary? These tests are all compiling today

**Comment by tmat** [comment]
test windows_debug_unit32_prtest please

**Comment by jaredpar** [comment]
Why the “#if” blocks. These shouldn’t be necessary.

**Comment by jaredpar** [comment]
Don't think these commits of mine should be cherry-picked. Several of them have already largely been undone by later changes in dev15.7.x. I'm going to send an email out shortly to swat to try and figure out why the the merge dev15.7.x to master isn't working.

**Comment by tmat** [comment]
Apparently the `#if`'s are currently needed: https://ci.dot.net/job/dotnet_roslyn/job/master/job/windows_debug_unit32_prtest/12372/

That indicates more commits are missing from master.

**Comment by jaredpar** [comment]
@tmat check your inbox. I'm not sure what is going on here or what we merged at this point.

**Comment by jaredpar** [comment]
@tmat we've all sync'd up. Proceed with the revert. The `#if` are necessary at this time given the current state of the branches.

**Comment by jaredpar** [comment]
retest windows_debug_unit64_prtest  please

**Comment by tmat** [comment]
test windows_debug_vs-integration_prtest please

**Comment by tmat** [comment]
test windows_release_vs-integration_prtest please

**Comment by tmat** [comment]
test windows_debug_vs-integration_prtest please

**Comment by jaredpar** [comment]
Looks like the windows_coreclr_test_prtest is in a zombie state. That job was renamed in a recent change to netci.groovy. Every other leg is passing. 

Let me know when you're ready to merge and I'll override the required flag for this leg.

**Comment by tmat** [comment]
@jaredpar I think it's good to go.

**Pull request merged**